src/imf_reader/sdr/read_announcements.py

```diff
@@ -54,7 +54,10 @@ def format_date(month: int, year: int) -> str:
 
 
 @lru_cache
-def get_holdings_and_allocations_data(year: int, month: int):
+def get_holdings_and_allocations_data(
+    month: int,
```

Comment:
@mharoruiz you accidentally switched the order of these parameters which are causing an issue further down the pipeline. The function`fetch_allocations_holdings` fails as a result. The tests are passing though. Please can you check this bug? 

The cache clearing functionality looks good though. Once the issue is addressed I can merge

Reply:
Sorry about that! I though it would usability but forgot to adjust the references down the line. It should be fixed now.